src/backend/marsha/core/tests/test_views_lti_video.py

```diff
@@ -60,15 +58,9 @@ def test_views_video_lti_post_instructor(self):
         data_jwt = match.group(1)
         jwt_token = AccessToken(data_jwt)
         self.assertEqual(jwt_token.payload["video_id"], str(video.id))
-        self.assertEqual(jwt_token.payload["user_id"], str(data.get("user_id")))
-        self.assertEqual(jwt_token.payload["context_id"], str(data.get("context_id")))
+        self.assertEqual(jwt_token.payload["user_id"], data.get("user_id"))
+        self.assertEqual(jwt_token.payload["context_id"], data.get("context_id"))
```

Comment:
actually in this case you can do data["user_id"] and data["context_id"]

applies several times in other diffs in the PR.